packages/hadron-typeorm/src/__tests__/connectionHelper.ts

```diff
@@ -62,6 +63,11 @@ describe('TypeORM connection helper', () => {
       assert(Container.take('teamRepository')),
     ));
 
+  it('should register UserStatus repository to container as userstatusRepository', () =>
```

Comment:
IMO it should be named `userStatusRepository`

Reply:
Why? According to the documentation no.
```
Repository keys in the container derive from names of schemas/classes and are built this way: <schema/class name in lower case>Repository

Examples:

User = userRepository
SuperUser = superuserRepository
loremIpsumDolor = loremipsumdolorRepository
```

Reply:
Do you prefer to leave the current solution and change documentation instead?

Reply:
I'm OK with leaving as it is